packages/astro/test/test-utils.js

```diff
@@ -103,8 +104,8 @@ export const silentLogging = {
 export async function loadFixture(inlineConfig) {
 	if (!inlineConfig?.root) throw new Error("Must provide { root: './fixtures/...' }");
 
-	/** @type {import('../src/core/logger/core').LogOptions} */
-	const logging = defaultLogging;
+	// Silent by default during tests to not pollute the console output
+	inlineConfig.logLevel = 'silent';
```

Comment:
Set the default logging to silent here, so most of the tests I've removed the manual `logging: silentLogging` config.